RustCrypto/RustCrypto.rs

```diff
@@ -0,0 +1,163 @@
+#![feature(test)]
+
+extern crate test;
+
+#[macro_use]
+extern crate crypto_bench;
+
+extern crate sha2;
+
+mod digest {
+    macro_rules! rust_crypto_digest_benches {
+        ( $name:ident, $block_len:expr, $output_len:expr, $digest:expr) => {
+            mod $name {
+                use sha2::Digest;
+                use crypto_bench;
+
+                digest_benches!($block_len, input, {
+                    let mut ctx = $digest;
+                    ctx.input(&input);
+                    let _ = ctx.result();
+                });
+            }
+        }
+    }
+/*
+    rust_crypto_digest_benches!(sha1, crypto_bench::SHA1_BLOCK_LEN,
+                                crypto_bench::SHA1_OUTPUT_LEN,
+                                sha1::Sha1::new());
+*/
```

Comment:
Why is this commented out? I think we should just remove it.

Reply:
I kept it for future contributors (or myself) as a reminder what to do next.
Removed the commented out code for now.